### DISCUSSION TITLE
Using projects [beta] - Can legacy projects be disabled?

### DISCUSSION THREAD

**Discussion by rbetts** [community/community#9017] [Projects and Issues]
The combination of both Projects and Projects [beta] creates some confusion. For example, we end up with duplicate projects or accidentally add issues to a legacy project when we intended projects beta.  Do the organization level project settings disable only the legacy projects? Can we configure the organization to use only projects [beta]?

**Comment by orcintre** [comment]
İlginize teşekkür ediyorum  ben anlamıyorum yapabilirseniz sevinirrim

17 Ara 2021 Cum, saat 20:37 tarihinde Ryan Betts ***@***.***>
şunu yazdı:

> The combination of both Projects and Projects [beta] creates some
> confusion. For example, we end up with duplicate projects or accidentally
> add issues to a legacy project when we intended projects beta. Do the
> organization level project settings disable only the legacy projects? Can
> we configure the organization to use only projects [beta]?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/github/feedback/discussions/9017>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVK65UJTXUR4N62JH7PF7FDURNYL5ANCNFSM5KJLMZGQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
orclntre
